src/convertToObject.js

```diff
@@ -6,7 +6,28 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceObject = {};
```

Comment:
The variable name 'sourceObject' could be more descriptive. Consider renaming it to reflect that it represents the styles as an object, such as 'stylesObject' or 'cssProperties'.